runtime-sdk/modules/evm/src/types.rs

```diff
@@ -116,3 +116,24 @@ mod eth {
     impl_upstream_conversions!(H160, H256, U256);
 }
 pub use eth::{H160, H256, U256};
+
+#[derive(Clone, Copy, Default, Eq, PartialEq)]
+pub struct StdU64 {
```

Comment:
Can you add a documentation comment? Also maybe call it something like `BlockRound` or `UInt64`. I guess we could also provide wrappers for all integers in the SDK so this wouldn't be needed here.

Reply:
Yes, if the SDK provide wrappers for all integers, this structure is not needed here.

Reply:
Perhaps it would be easier to use the `construct_uint!` macro to generate this type alongside the ones already defined above.